deploy/kubernetes/seqr/seqr.gcloud.yaml

```diff
@@ -127,6 +127,11 @@ spec:
             secretKeyRef:
               name: seqr-secrets
               key: google_client_secret
+        - name: SERVICE_ACCOUNT_FOR_ANVIL
+          valueFrom:
+            secretKeyRef:
+              name: seqr-secrets
+              key: anvil_service_account_id
```

Comment:
you need to update the secrets deployment script to include this

Reply:
The file `deploy/servctl_utils/deploy_command_utils.py` is updated now.